src/app/client/src/app/app.component.ts

```diff
@@ -121,6 +121,36 @@ export class AppComponent implements OnInit, OnDestroy {
       });
   }
   ngOnInit() {
+    this.activatedRoute.queryParams.subscribe(params => {
+      if (params && (params['utm_campaign'] || params['utm_medium'] || params['utm_source']
+      || params['utm_term'] || params['utm_content'])) {
```

Comment:
Can we try using
```javascript
const params = {
    utm_campaign: '__utm_campaign',
    utm_medium: '__ utm_medium',
    utm_source: '__ utm_course',
    utm_term: '__utm_term',
    utm_content: '__utm_content'
}

const keysExpected = ['utm_term', 'NON_EXISTS', 'utm_source'];

console.log(_.some(_.intersection(keysExpected, _.keys(params))))
```

CC: @rajeevsathish

Reply:
Thanks Rajesh I had implemented this one.